packages/cozy-stack-client/src/__tests__/__snapshots__/OAuthClient.spec.js.snap

```diff
@@ -0,0 +1,159 @@
+
+exports[`OAuthClient with registration should throw when refreshing the access token without token 1`] = `"No token to refresh"`;
+
+exports[`OAuthClient with registration should throw when the provided access code is different from the URL 1`] = `"Given state does not match url query state"`;
+
+exports[`OAuthClient with registration should throw when trying to register again 1`] = `"Client already registered"`;
+
+exports[`OAuthClient with registration should unregister a client 1`] = `
+Array [
+  "http://cozy.tools:8080/auth/register/1",
+  Object {
+    "body": "null",
+    "credentials": "include",
+    "headers": Object {
+      "Accept": "application/json",
+      "Authorization": "Bearer 1234",
+      "Content-Type": "application/json",
+    },
+    "method": "DELETE",
+  },
+]
+`;
+
+exports[`OAuthClient with registration should update client informations 1`] = `
+Array [
+  "http://cozy.tools:8080/auth/register/1",
+  Object {
+    "body": "{\\"client_id\\":\\"1\\",\\"client_name\\":\\"TestClientName\\",\\"redirect_uris\\":[\\"http://localhost\\"],\\"software_id\\":\\"TestSofwareID\\",\\"policy_url\\":\\"http://example.com\\"}",
+    "credentials": "include",
+    "headers": Object {
+      "Accept": "application/json",
+      "Authorization": "Bearer 1234",
+      "Content-Type": "application/json",
+    },
+    "method": "PUT",
+  },
+]
+`;
+
+exports[`OAuthClient without registration should not generate a auth code URL 1`] = `"Client not registered or missing OAuth informations"`;
+
+exports[`OAuthClient without registration should register a client 1`] = `
+Array [
+  "http://cozy.tools:8080/auth/register",
+  Object {
+    "body": "{\\"redirect_uris\\":[\\"http://localhost\\"],\\"client_name\\":\\"TestClientName\\",\\"software_id\\":\\"TestSofwareID\\",\\"client_kind\\":\\"\\",\\"client_uri\\":\\"\\",\\"logo_uri\\":\\"\\",\\"policy_uri\\":\\"\\",\\"software_version\\":\\"\\",\\"notification_platform\\":\\"\\",\\"notification_device_token\\":\\"\\"}",
+    "headers": Object {
+      "Accept": "application/json",
+      "Content-Type": "application/json",
+    },
+    "method": "POST",
+  },
+]
+`;
+
+exports[`OAuthClient without registration should throw on other server interactions 1`] = `"Client not registered or missing OAuth informations"`;
+
+exports[`OAuthClient without registration should throw on other server interactions 2`] = `"Client not registered or missing OAuth informations"`;
+
+exports[`OAuthClient without registration should throw on other server interactions 3`] = `"Client not registered or missing OAuth informations"`;
+
+exports[`OAuthClient without registration should throw on other server interactions 4`] = `"Client not registered or missing OAuth informations"`;
+
+exports[`OAuthClient without registration should throw on other server interactions 5`] = `"Client not registered or missing OAuth informations"`;
```

Comment:
Do you check the content of snapshots? I can't review it, but I think it is a good idea to review it carefuly. So if you could throw a double attention here as we will be ablt to rely on those snapshots later.

Reply:
Yes :) I checked them as I was writing the tests and actually caught a few problems with them.